bundle.yaml

```
@@ -36,6 +36,21 @@ applications:
     channel: "stable"
     scale: 1
 
+  nginx-ingress-integrator-sdlc:
```

```suggestion
  sdlc-ingress:
```

We could consider some shorter names here?

---

On this, we could rename a bunch of these charms so you get:

- mongodb
- legend-studio
- legend-sdlc
- legend-engine
- gitlab-integrator
- legend-db

Just makes typing juju commands a bit shorter! Thoughts @pedroleaoc / @balbirthomas ?

---

- Sounds good to me but we will need to change all the charms also (right ?). 
- What about "k8s" suffix (since these are k8s charms) ?

---

No, changing the name at the top level in the bundle is just like deploying a charm with an alias: `juju deploy mongodb-k8s mongodb`, for example

---

Done